CMake/AddShader.cmake

```diff
@@ -2,14 +2,15 @@
 include(GetRelativePath)
 
 function(add_shader RET)
-	# Find glslc shader compiler.
-	# On Android, the NDK includes the binary, so no external dependency.
-	if(ANDROID)
-		file(GLOB glslc-folders ${ANDROID_NDK}/shader-tools/*)
-		find_program(GLSLC glslc HINTS ${glslc-folders})
-	else()
-		find_program(GLSLC glslc)
-	endif()
+
+    # add_shader depends on Vulkan::glslc. if not found, guide the user to find vulkan and the executable.
+    if(NOT Vulkan_FOUND)
+        message(FATAL_ERROR
+            "The addShader() function depends on the "glslc" shader compiler executable.\n"
+            "It is detected during find_package(VULKAN REQUIRED) and defined as imported target executable Vulkan::glslc.\n"
+            "Please use find_package(VULKAN REQUIRED) and make sure it succeeds!\n"
```

Comment:
Shouldn't that be: find_package(Vulkan REQUIRED)

It seems the first argument for find_package is case sensitive, when I changed it to "vulkan" yesterday it complained about that it should be "Vulkan".

Reply:
You are right.. change incomming.